plugins/github-actions/src/components/useProjectName.ts

```diff
@@ -13,15 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-import useAsync from 'react-use/lib/useAsync';
 import { Entity } from '@backstage/catalog-model';
 
 export const GITHUB_ACTIONS_ANNOTATION = 'github.com/project-slug';
 
-export const useProjectName = (entity: Entity) => {
-  const { value, loading, error } = useAsync(async () => {
-    return entity?.metadata.annotations?.[GITHUB_ACTIONS_ANNOTATION] ?? '';
-  });
-  return { value, loading, error };
-};
+export const useProjectName = (entity: Entity) =>
```

Comment:
Tbh this is no longer a hook, so we could just call it `getProjectNameFromEntity` or something like that, to avoid confusion

Reply:
Good point! Done!